cilium-dbg/cmd/service_update.go

```diff
@@ -120,6 +120,10 @@ func updateService(cmd *cobra.Command, args []string) {
 	warnIdTypeDeprecation()
 
 	id := int64(idU)
+	if id > 65535 {
```

Comment:
I would be good to reference a const or use the `MaxSetOfServiceID` as we've done below. 
Also prefer referencing that limit in the error message vs hard coding it.

Reply:
Done